changes-set.txt

```diff
@@ -99,6 +99,7 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+19-Aug-24 rabeqdca  [same]	moved from SN's mathbox to main set.mm
```

Comment:
typo, rabeq**cd**a, but note that #4139 will create a merge conflict anyway

Reply:
Fixed. I'll do the fixup once #4139 is merged.